docs/source/tor.rst

```diff
@@ -1,7 +1,9 @@
 Connecting to Tor
 =================
 
-Pick a way to connect OnionShare to Tor by clicking the Tor onion icon in the bottom right of the OnionShare window to open the Tor Settings tab.
+.. _tor:
+
+Pick a way to connect OnionShare to Tor by clicking Network Settings from the welcome screen, or the Tor onion icon in the bottom right of the OnionShare window to open the Tor Settings tab.
```

Comment:
Now there is no longer a separate Tor onion icon in the bottom right. Now everyone needs to click the settings icon and then go to the "Tor Settings" subtab.